site/src/playground/index.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: default
+title: Playground Gallery
+---
+<div class="previews container">
```

Comment:
Can you add a title and a few lines of text to this page to describe what it is?